samples/estimation/df-chemistry/chemistry.py

```diff
@@ -488,11 +487,18 @@ def ndarray2d_to_string(arr):
 
 # Collect resource estimation parameters
 if args.paramsfile is None:
-    params = {
-        "errorBudget": 0.01,
-        "qubitParams": {"name": "qubit_maj_ns_e6"},
-        "qecScheme": {"name": "floquet_code"},
-    }
+    params = [
+        {
+            "errorBudget": 0.01,
+            "qubitParams": {"name": "qubit_maj_ns_e6"},
+            "qecScheme": {"name": "floquet_code"},
+        },
+        {
```

Comment:
These two sets of parameters look to be the same, it would make sense to make them different if we want to illustrate running resource estimates on different parameters at once

Reply:
Removed the extra example. I think the original file had only one.
In case you have any specific second example in mind, I can add that here.

Reply:
I don't have a specific example in mind, but it makes sense that if we're updating the code to run multiple estimates at once and print the summary table, that table has several rows. Any other built-in qubit platform would work here

Reply:
Sure, I updated the code with another example.